src/modinfodialogconflicts.cpp

```diff
@@ -723,6 +735,68 @@ ConflictItem GeneralConflictsTab::createOverwrittenItem(
     index, std::move(fileName), true, std::move(altOrigin), archive);
 }
 
+QString percent(int a, int b) {
+  if (b == 0) {
+    return QString::number(0, 'f', 2);
```

Comment:
This should return something like `-` or `N/A`. Showing 0% of 0 doesn't make much sense.

Reply:
Do you really care about this, because I have "%" in the table and I would need to remove them all and move them here to be able to show "-" instead of "-%"

Reply:
Don't care.